code/style.css

```diff
@@ -2,6 +2,327 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+
+
+/* for smooth scrolling when pressing the animation */
+html{
+    scroll-behavior: smooth;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    max-width: 100vw;
+}
+
+
+/* Hero image and text */
+.hero {
+    background: url('https://images.unsplash.com/photo-1621605815971-fbc98d665033?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80') center;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    max-width: 100vw;
+    height: 100%;
+    overflow-x: hidden;
+}
+
+.bigtext {
+    font-size: 10vw;
+    padding-bottom: 1rem;
+    color: #EFEEED;
+    text-decoration: underline;
+}
+
+/* The div around the animation */
+.scroll-down {
+    height: 50px;
+    width: 30px;
+    border: 2px solid orange;
+    position: absolute;
+    bottom: 20px;
+    border-radius: 50px;
+    cursor: pointer;
+    display: flex;
+    align-content: center;
+}
+/* I had problems centering it without destroying the animation it is 1% off, live with it please :> */
+.halfling {
+    left: 49%;
+    margin: auto;
+}
+
+/* the animation and keyframes */
+.scroll-down::before,
+.scroll-down::after {
+    content: "";
+    position: absolute;
+    top: 20%;
+    left: 50%;
+    height: 10px;
```

Comment:
it's super interresting to see how you did this! I get inspired to try it next time aswell. even though you thought it was hell 😃